components/fonts/platform/freetype/android/xml.rs

```diff
@@ -41,7 +41,7 @@ pub(super) fn parse(bytes: &[u8]) -> xml::reader::Result<Vec<Node>> {
                     nodes.push(Node::Text(s))
                 }
             },
-            StartDocument { .. } | EndDocument | ProcessingInstruction { .. } | Comment(..) => {},
+            StartDocument { .. } | EndDocument | ProcessingInstruction { .. } | Comment(..) => {}, _ => {},
```

Comment:
Instead of listing half of the variants and then a wildcard, please add the new variant here. So instead of `_`, write `Doctype`.